dojo/tools/blackduck/importer.py

```diff
@@ -0,0 +1,78 @@
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from pathlib import Path
```

Comment:
This relies on some 3.6 features, though the class was added earlier than that. Basically, a lot of standard library IO calls assume the path parameter is a string or bytes, and PEP 519 cleans up this whole mess by standardizing a `PathLike` class which effectively allows for strings, bytes, or `Path` instances, to all be valid parameters to those IO functions.

To work around this, you can wrap the path in `string(path)` to convert it at the call site.